packages/mobile/src/components/core/GradientIcon.tsx

```diff
@@ -0,0 +1,31 @@
+import { ComponentType } from 'react'
```

Comment:
Note: https://github.com/AudiusProject/audius-client/blob/main/packages/mobile/src/components/enable-push-notifications-drawer/EnablePushNotificationsDrawer.tsx#L145

I believe this does what you're doing here.
I like GradientIcon though.

Reply:
> Realize now, we already have this drawer
> 
> https://github.com/AudiusProject/audius-client/blob/main/packages/mobile/src/components/mobile-upload-drawer/MobileUploadDrawer.tsx#L29

Lol im so dumb! took not that long at least :D will merge them